### PR TITLE
feat: adds folder for aws lambda layers

### PR DESCRIPTION
**Pull request by scefali**
This PR shows how we would add the AWS Lambda Layer information to the release registry. We will need this information in our docs (manual setup) as well as in Sentry (automated setup). AWS Lambda layers are installed by providing the ARN of the layer (which can be done manually or automatically with the AWS Lambda integration that is coming out). The layer ARN has the following components:
1. Account number
2. Region (user must pick the region they are in)
3. Layer name
4. Layer version

The layer version is an auto-incrementing number that would be updated each time we make a new release for that SDK. We should also keep track of what SDK version maps to what version of the layer in case someone needs to debug an older layer.

Note that I only include the `1.json` and `latest.json` here to show what it would look like. Craft should be always generating these files during the release process.

### CODE REVIEW

**Comment by HazAT** [comment]
@scefali I wonder if it would make sense just to add the new fields to the already existing entry in the registry and not create a new folder

With that we could make sure that the version we released to npm is the version on aws lambda

**Comment by scefali** [comment]
@HazAT Great question. My concern of just putting the new fields in the existing entry has to do with the fact that the Python layer we publish is runtime specific (see https://github.com/getsentry/sentry-python/pull/927/files). We have a different Layer for each version of Python. So a single Python release would have four new layer versions. I'm also conerned about adding a bunch of new fields to an NPM release that have nothing to do with NPM and I'm worried it might make it harder to figure out which versions of the Layer we've released (some releases might have the new fields but others won't) A separate folder just seems cleaner IMO. What's the hesitation of having it in a new folder? Would it help if I put it somewhere like the `misc` directory?

**Comment by HazAT** [comment]
OK, I guess it makes sense. It's no problem having another folder I just thought it's a 1:1 relation to the SDK.

File: aws-lambda-layers/node/1.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "AWS Lambda Node Layer",
+  "canonical": "aws-layer:node",
+  "version": "1",
+  "sdk_version": "5.29.2",
+  "account_number": "943013980633",
+  "layer_name": "SentryNodeServerlessSDK",
+  "repo_url": "https://github.com/getsentry/sentry-javascript",
+  "main_docs_url": "https://docs.sentry.io/platforms/node/guides/aws-lambda"
```
Comment (scefali) [inline aws-lambda-layers/node/1.json:9]: we should add a list of regions

File: aws-lambda-layers/node/latest.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (rhcarvalho) [inline aws-lambda-layers/node/latest.json:1]: IIRC this file should be a symlink

Comment (rhcarvalho) [reply aws-lambda-layers/node/latest.json:1]: see https://github.com/getsentry/sentry-release-registry/blob/master/bin/sync-links

**Pull request closed without merging**